packages/databricks-vscode/src/bundle/AuthenticatedBundleModel.ts

```diff
@@ -0,0 +1,81 @@
+import {Disposable, Uri} from "vscode";
+import {BundleWatcher} from "./BundleWatcher";
+import {AuthProvider} from "../configuration/auth/AuthProvider";
+import {Mutex} from "../locking";
+import {CliWrapper} from "../cli/CliWrapper";
+import {BundleTarget} from "./types";
+import {CachedValue} from "../locking/CachedValue";
+import {onError} from "../utils/onErrorDecorator";
+import lodash from "lodash";
+import {workspaceConfigs} from "../vscode-objs/WorkspaceConfigs";
+
+export class AuthenticatedBundleModel implements Disposable {
```

Comment:
It's sort of make sense for the AuthenticatedBundleConfigLoader to be named that way (although I prefer `configuration/AuthenticatedBundleConfigModel`), since it explicitly filters out all keys that are not related to the auth. But this one doesn't filter anything, should we just name it `BundleModel`? We can think about better names when (of if) we add more bundle-related models (e.g. pre auth or post deploy that you mentioned).

Reply:
I believe you will be happy with the new changes :)